src/styles/modules/_m-payment-form.scss

```diff
@@ -58,6 +58,10 @@
     &:hover {
       background-color: black;
     }
+
+    @include breakpoint(tablet-plus) {
+      width: 66%;
```

Comment:
This can actually be done with the columns mixin so it has the correct column spacing and gutters: `@include column(4/12);` Give this a try and see if it looks right to you.